ambassador/ambassador/config/config.py

```diff
@@ -372,19 +385,25 @@ def validate_object(self, resource: ACResource) -> RichStatus:
             apiVersion = apiVersion.replace('getambassador.io/', 'ambassador/')
             resource.apiVersion = apiVersion
 
+        is_ambassador = False
+
         # Ditch the leading ambassador/ that really needs to be there.
         if apiVersion.startswith("ambassador/"):
+            is_ambassador = True
             apiVersion = apiVersion.split('/')[1]
+        elif apiVersion.startswith('serving.knative.dev') or apiVersion.startswith('networking.internal.knative.dev'):
+            pass
         else:
             return RichStatus.fromError("apiVersion %s unsupported" % apiVersion)
 
         version = apiVersion.lower()
 
         # Is this deprecated?
-        status = Config.SupportedVersions.get(version, 'is totally bogus')
+        if is_ambassador:
+            status = Config.SupportedVersions.get(version, 'is totally bogus')
 
-        if status != 'ok':
-            self.post_notice(f"apiVersion {originalApiVersion} {status}", resource=resource)
+            if status != 'ok':
+                self.post_notice(f"apiVersion {originalApiVersion} {status}", resource=resource)
```

Comment:
Whoa. I seriously wrote code so that a customer could see "apiVersion 99.3 is totally bogus"?? OMG. Let's, uh, fix that? 🙂

Reply:
Hehe. Changed it to "is not supported"

```
            status = Config.SupportedVersions.get(version, 'is not supported')
```